src/renderer/components/+custom-resources/crd-resources.tsx

```diff
@@ -91,10 +92,15 @@ export class CrdResources extends React.Component<Props> {
         renderTableContents={(crdInstance: KubeObject) => [
           crdInstance.getName(),
           isNamespaced && crdInstance.getNs(),
-          ...extraColumns.map(column => ({
-            renderBoolean: true,
-            children: jsonPath.value(crdInstance, column.jsonPath.slice(1)),
-          })),
+          ...extraColumns.map((column) => {
+            const pathExpression = parseJsonPath(column.jsonPath.slice(1));
+
+            return {
+              renderBoolean: true,
+              children: jsonPath.value(crdInstance, pathExpression),
+            };
+          }
+          ),
```

Comment:
```suggestion
          ...extraColumns.map(column => ({
            renderBoolean: true,
            children: parseJsonPath(
              jsonPath.value(crdInstance, column.jsonPath.slice(1))
            ),
          })),
```

Reply:
parseJsonPath must be done before calling `jsonPath.value` because that is the call that is throwing error. Changed my implementation to one-liner.